apps/avifenc.c

```diff
@@ -66,22 +66,14 @@ static void syntax(void)
     printf("                                        M = matrix coefficients\n");
     printf("                                        (use 2 for any you wish to leave unspecified)\n");
     printf("    -r,--range RANGE                  : YUV range [limited or l, full or f]. (JPEG/PNG only, default: full; For y4m or stdin, range is retained)\n");
-    printf("    --min Q                           : Set min quantizer for color (%d-%d, where %d is lossless)\n",
-           AVIF_QUANTIZER_BEST_QUALITY,
-           AVIF_QUANTIZER_WORST_QUALITY,
-           AVIF_QUANTIZER_LOSSLESS);
-    printf("    --max Q                           : Set max quantizer for color (%d-%d, where %d is lossless)\n",
-           AVIF_QUANTIZER_BEST_QUALITY,
-           AVIF_QUANTIZER_WORST_QUALITY,
-           AVIF_QUANTIZER_LOSSLESS);
-    printf("    --minalpha Q                      : Set min quantizer for alpha (%d-%d, where %d is lossless)\n",
-           AVIF_QUANTIZER_BEST_QUALITY,
-           AVIF_QUANTIZER_WORST_QUALITY,
-           AVIF_QUANTIZER_LOSSLESS);
-    printf("    --maxalpha Q                      : Set max quantizer for alpha (%d-%d, where %d is lossless)\n",
-           AVIF_QUANTIZER_BEST_QUALITY,
-           AVIF_QUANTIZER_WORST_QUALITY,
-           AVIF_QUANTIZER_LOSSLESS);
+    printf("    --q Q                             : Set quality for color (%d-%d, where %d is lossless)\n",
```

Comment:
`--q` does not override `--min` & `--max`, but rather it changes the default values of `--min` & `--max`. Since the help message printed by `syntax()` does not document the default values of `--min` & `--max`, I did not describe this behavior.

Do you think I should?

Reply:
James: I just realized my previous comment is wrong. Please ignore it. Here is the correct reply.

`--q` does not override `--min` & `--max`. But the default value of `--q` depends on whether `--min` & `--max` are specified.

Since the help message (printed by `syntax()`) does not document the default values of `--min` & `--max`, it does not document the default value of `--q` either.

Do you think it should?